core/templates/dev/head/domain/skill/SkillUpdateService.js

```diff
@@ -115,6 +119,34 @@ oppia.factory('SkillUpdateService', [
           });
       },
 
+      setSupersedingSkillId: function(skill, newSupersedingSkillId) {
```

Comment:
For this and the following function--are these necessary? I don't seem to see them being used anywhere. (I also forgot the purpose of keeping track of the superseding skill id)